### PR TITLE
Reorganize readme

### PR DESCRIPTION
**Pull request by six5532one**
Changes:
* Other developers and I looked for the public documentation because it wasn't very discoverable. Even though the README links to it, I missed these links in the README during my onboarding. To make the public docs more discoverable, this PR adds a dedicated section for it near the beginning of the README.
* To prevent discouraging new users, the README should make it easy for them to start using the library. Some folks may get discouraged after running into installation issues and abandon the effort. When I got started, I found using the container to be much less frictionless. This PR organizes the Quick Start section to make it more obvious up front that users can interact with the library with a container before investing in installing it.
* Just speaking to my experience, the table in the Quick Start section distracted me from getting started. The "strategies", "table support" and "options" confused me because I didn't have enough context yet. In the spirit of keeping Quick Start focused on just getting some running code, this PR moves the table to Quick Tour, instead. 
* I assume more readers are interested in using the library than developing it so this PR places the Quick Tour above the instructions for developing the library.
~~* Mentions One Drive connector in public docs~~ (done in https://github.com/Unstructured-IO/unstructured/pull/949)

... screenshot truncated the instructions for using the container ...

... screenshot truncated the table of file types ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,10 +82,9 @@ installation. NOTE: We do not currently support python 3.11, please use an older
     - `poppler-utils` (images and PDFs)
     - `tesseract-ocr` (images and PDFs)
     - `libreoffice` (MS Office docs)
-
-- If you are parsing PDFs and want to use a model from the [layoutparser model
-  zoo](https://github.com/Unstructured-IO/unstructured-inference#using-models-from-the-layoutparser-model-zoo),
-  use the instructions [here](https://github.com/Unstructured-IO/unstructured-inference#detectron2).
```
Comment (six5532one) [inline README.md:80]: In the interest of keeping the README focused, this was removed in favor of linking to the public installation docs, where all details can be found.

File: README.md [hunk 2]
```diff
@@ -88,41 +95,10 @@ elements = partition(filename="example-docs/eml/fake-email.eml")
...
-| Power Points (`.pptx`) | `partition_pptx` | N/A | Yes | Include Page Breaks |
-| ReStructured Text (`.rst`) | `partition_rst` | N/A | Yes | Include Page Breaks |
-| Rich Text Files (`.rtf`) | `partition_rtf` | N/A | Yes | Include Page Breaks |
-| TSV Files (`.tsv`) | `partition_tsv` | N/A | Yes | None |
-| Word Documents (`.doc`) | `partition_doc` | N/A | Yes | None |
-| Word Documents (`.docx`) | `partition_docx` | N/A | Yes | None |
-| Word Documents (`.doc`) | `partition_doc` | N/A | Yes | Include Page Breaks |
-| Word Documents (`.docx`) | `partition_docx` | N/A | Yes | Include Page Breaks |
-| XML Documents (`.xml`) | `partition_xml` | N/A | No | Encoding; Max Partition; XML Keep Tags |
-
```
Comment (six5532one) [inline README.md:122]: This is replaced with a link to Quick Tour, which includes more context before presenting this table.

File: README.md [hunk 3]
```diff
@@ -170,39 +146,6 @@ python3
 >>> elements = partition_text(filename="example-docs/fake-text.txt")
 ```
 
-
-## :coffee: Installation Instructions for Local Development
```
Comment (six5532one) [inline README.md:174]: This is moved below Quick Tour so that readers can more quickly find information about using the library.

File: README.md [hunk 4]
```diff
@@ -215,14 +158,9 @@ of the features in the library.
 
 ### Document Parsing
 
-The easiest way to parse a document in unstructured is to use the `partition` brick. If you
-use `partition` brick, `unstructured` will detect the file type and route it to the appropriate
-file-specific partitioning brick.
```
Comment (six5532one) [inline README.md:220]: >If you
use `partition` brick, `unstructured` will detect the file type and route it to the appropriate
file-specific partitioning brick.

This is moved below the code snippet, where it immediately prefaces and contextualizes the table that displays supported file types and their corresponding partition function.

File: docs/source/installing.rst [hunk 5]
```diff
@@ -5,17 +5,22 @@ Quick Start
 -----------
 
 Use the following instructions to get up and running with ``unstructured`` and test your
-installation.
+installation. NOTE: We do not currently support python 3.11; please use an older version.
```
Comment (six5532one) [inline docs/source/installing.rst:8]: @MthwRobinson is this no longer the case, per https://github.com/Unstructured-IO/unstructured/pull/908? If so, I'll remove from here and the README

Comment (MthwRobinson) [reply docs/source/installing.rst:8]: Correct, the `unstructured` library now supports 3.11. You can drop that line from the docs and readme.

**Comment by six5532one** [comment]
closing in favor of https://github.com/Unstructured-IO/unstructured/pull/962

**Pull request closed without merging**